examples/causal_inference/interventional_distribution.ipynb

```diff
@@ -0,0 +1,1561 @@
+{
```

Comment:
<p>I'd maybe sign-post a bit more about why this will be important. You don't want to assume the reader has already read the blog-post or that they need to go read the blog post to understand what's happening next. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Done</p>